integration-tests/package.json

```diff
@@ -8,21 +8,21 @@
     "test-flakes": "FLAKES=true yarn test-local --runInBand"
   },
   "devDependencies": {
-    "jest": "^27.4.3",
-    "jest-environment-puppeteer": "^6.0.3",
+    "jest": "^29.7.0",
+    "jest-environment-puppeteer": "^10.1.1",
     "jest-junit": "^13.0.0",
-    "jest-puppeteer": "^6.0.2",
+    "jest-puppeteer": "^10.1.1",
     "node-fetch": "^2.6.7",
     "prompts": "^2.4.1",
-    "puppeteer-cluster": "^0.23.0"
+    "puppeteer-cluster": "^0.24.0"
   },
   "dependencies": {
     "@axe-core/puppeteer": "^4.6.0",
     "date-fns": "^2.24.0",
     "google-auth-library": "^7.9.2",
     "lodash": "^4.17.21",
     "p-retry": "^4.6.1",
-    "puppeteer": "^13.1.3",
+    "puppeteer": "22.7.1",
```

Comment:
This is the last Puppeteer version that works with the IA iframes (using Chrome version 124.0.6367.78). All later versions I tried failed in the same way: broken computer icon in the iframe and a message about being unable to access dev leonardo.

Reply:
@jdcanas just pinging you in case you might know, but it sounds like we eventually will need to rewrite our IA UI Integration tests to be better compatible with pupeteer?